azext_iot/tests/iothub/devices/test_iothub_nested_edge_int.py

```diff
@@ -195,6 +196,9 @@ def test_iothub_nested_edge(self):
                 checks=self.is_empty(),
             )
 
+            # Wait for child devices to be removed to prevent failures
+            sleep(60)
```

Comment:
60 seconds for this seems a long time, does shortening it to 15-30 seconds have failures?

Reply:
yeah 30 seconds was causing intermittent failures.